samples/src/baremetal-semihosting/Makefile

```diff
@@ -20,7 +20,7 @@ include ../../Makefile.conf
 build: hello.elf
 
 hello.elf: *.c
-	$(BIN_PATH)/clang --config armv6m_soft_nofp_semihost_baremetal -g -T ../../ldscripts/microbit.ld -o hello.elf ../../startup/startup_ARMCM0.S $^
+	$(BIN_PATH)/clang --config armv6m_soft_nofp_semihost -g -T ../../ldscripts/microbit.ld -o hello.elf ../../startup/startup_ARMCM0.S $^
 
```

Comment:
Do we still need the "-m 1073741824k" workaround in the run command with picolibc?

Reply:
We do not :)